### PR TITLE
MineMeld treat properly non root server URLs

### PR DESCRIPTION
**Pull request by yardensade**
Also changed descriptions of DBOTSCORE output paths according to our new standards.

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18898

## Description
weird behavior of urllib2, was omitting the subdomain from URLs on join operations (concat server root + api path) for instances that were installed on non root paths in server.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Code Review

### CODE REVIEW

**Comment by yardensade** [comment]
Do not merge this until the DBotScore pre commit hook is sorted, had to skip hooks for now.

**Comment by yardensade** [comment]
Passed client hooks, now hoping nothing broke in the process.

**Comment by Itay4** [comment]
@kirbles19 please review the changelog

File: Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml [hunk 1]
```diff
@@ -383,24 +383,24 @@ script:
       description: url
       isArray: false
       name: url
-      required: true
+      required: false
```
Comment (Itay4) [inline Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml]: change this back to be `true` 
the file validation was fix here https://github.com/demisto/content/pull/4255

File: Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml [hunk 2]
```diff
@@ -308,24 +308,24 @@ script:
       description: domain
       isArray: false
       name: domain
-      required: true
+      required: false
```
Comment (Itay4) [inline Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml]: change this back to be `true` 
the file validation was fix here https://github.com/demisto/content/pull/4255

File: Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml [hunk 3]
```diff
@@ -227,24 +227,24 @@ script:
       description: file
       isArray: false
       name: file
-      required: true
+      required: false
```
Comment (Itay4) [inline Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml]: change this back to be `true` 
the file validation was fix here https://github.com/demisto/content/pull/4255

File: Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml [hunk 4]
```diff
@@ -152,24 +152,24 @@ script:
       description: ip
       isArray: false
       name: ip
-      required: true
+      required: false
```
Comment (Itay4) [inline Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml]: change this back to be `true` 
the file validation was fix here https://github.com/demisto/content/pull/4255

File: Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.py [hunk 5]
```diff
@@ -63,7 +62,7 @@ def _call_api(self, uri, data=None, headers=None, method=None):
         if headers is None:
             headers = {}
 
-        api_url = urljoin(self.url, uri)
+        api_url = "".join([self.url, uri])
```
Comment (Itay4) [inline Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.py]:
```suggestion
        api_url = ''.join([self.url, uri])
```

File: Integrations/PaloAlto_MineMeld/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+  - Added support to non root URL structures.
```
Comment (yaakovi) [inline Integrations/PaloAlto_MineMeld/CHANGELOG.md]:
```suggestion
Added support to non root URL structures.
```

**Comment by kirbles19** [comment]
@yardensade Done.

**Pull request merged**